pkg/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -868,20 +881,34 @@ func (v *vpnSeedServer) emptyEnvoyFilter() *networkingv1alpha3.EnvoyFilter {
 	return &networkingv1alpha3.EnvoyFilter{ObjectMeta: metav1.ObjectMeta{Name: v.namespace + "-vpn", Namespace: v.istioNamespaceFunc()}}
 }
 
+//TODO: unused function?
```

Comment:
?

Reply:
The below function is unused just an FYI